CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ set( PYIODACONV_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/p
 ## Dependencies
 
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+find_package( pybind11 REQUIRED)
```

Comment:
how much effort is it to keep this optional and only build this if it is found? Not sure about everyone's thoughts on adding a dependency. This isn't 'required' (but optional) for IODA I don't think...

Reply:
Ok, I made pybind11 optional. I followed the existing patterns. The BUFR python component will only be built if the bufr component is built and pybind11 is found.